protos/sources/data-node/api/v2/trading_data.proto

```diff
@@ -3456,3 +3477,134 @@ message LiquidationPrice {
   // Liquidation price assuming sell orders start getting filled.
   string including_sell_orders = 3;
 }
+
+// Team record containing the team information.
+message Team {
+  // Unique identifier of the created team.
+  string team_id = 1;
+  // Party ID that created the team.
+  string referrer = 2;
+  // Optional team name to be added to the referral banner.
```

Comment:
Maybe minor: I don't know what the "referral banner" is and wonder if its specific to how _our_ FE is going to use the API. Shouldn't the description of the field just be:
```
// Name of the team
```
?

Also we're not saying what it means if the team name is empty, from my understanding of the spec it means "the team is not participating in any leaderboards".

Similar for the other optional fields:
```
link to an image to be used as the team avatar
->
link to an image of the team's avatar

link to a team forum, Discord, etc.
->
link to the team's homepage
```
?

Reply:
I have no idea, that piece of documentation was how it is referred to in the original CreateTeam event proto. @ValentinTrinque, any insights?

Reply:
It seems sensible to me. I just used the description from the spec. I think we can change this in both proto.

Reply:
I will address and make appropriate adjustments to the documentation as suggested.